python/proton/_message.py

```diff
@@ -515,18 +515,20 @@ def _set_annotations(self, annotations):
 
     def encode(self):
         self._pre_encode()
+        bytes = ffi.new("char *")
```

Comment:
The size of `bytes` has to be `sz`. So, looking at https://cffi.readthedocs.io/en/latest/using.html

```python
>>> import cffi.api
>>> ffi = cffi.api.FFI()
>>> bytes = ffi.new("char []", 10)
<cdata 'char[]' owning 10 bytes>
```

So first you initialize `sz`, then you allocate `data` of size `sz`.

Reply:
btw, don't use the import the way I do, what you do there is ok; I just wanted to have the ffi object in terminal without building the binding itself, so i could do a demonstration